src/runtime/storageNG/tests/store-test.ts

```diff
@@ -186,6 +188,7 @@ describe('Store', async () => {
     const count = new CRDTCount();
     count.applyOperation({type: CountOpTypes.Increment, actor: 'me', version: {from: 0, to: 1}});
 
+    // eslint-disable-next-line no-async-promise-executor
     return new Promise(async (resolve, reject) => {
```

Comment:
Here too async + await can be removed without changing the logic, I think... Do that instead of disabling the eslint check?

Reply:
Indeed.  remove() returns a promise so it can just be await'd